libs/search-widget/src/components/modal/Modal.svelte

```diff
@@ -100,11 +100,25 @@
     max-width: 100vw;
     background-color: transparent;
   }
-  :not(.transparent) .modal-content {
+  .modal:not(.transparent) .modal-content {
     background-color: var(--color-light-stronger);
     box-shadow: var(--shadow-modal);
   }
 
+  .modal.transparent .modal-content {
+    overflow: auto;
+    padding-right: 16px;
+  }
+  .modal.transparent .modal-content::-webkit-scrollbar {
+    width: 6px;
+  }
+  .modal.transparent .modal-content::-webkit-scrollbar-thumb {
+    background-color: #000;
```

Comment:
use a `var` (define new ones in `wdiget.svelte` if this color is not existing yet)

Reply:
fixed